distage/distage-framework-docker/src/main/scala/izumi/distage/docker/examples/ZookkeeperDocker.scala

```diff
@@ -1,23 +1,29 @@
 package izumi.distage.docker.examples
 
+import java.util.UUID
+
 import distage.{ModuleDef, TagK}
-import izumi.distage.docker.ContainerDef
 import izumi.distage.docker.Docker.{ContainerConfig, ContainerNetwork, DockerPort}
+import izumi.distage.docker.{ContainerDef, DockerClientWrapper, DockerContainer}
+import izumi.distage.model.effect.{DIEffect, DIEffectAsync}
+import izumi.logstage.api.IzLogger
 
 object ZookeeperDocker extends ContainerDef {
   val zookeeperNetwork: ContainerNetwork = ContainerNetwork("zookeeper-network")
   override def config: Config = {
     ContainerConfig(
       image = "zookeeper:3.4.13",
       ports = Seq(DockerPort.TCP(2181), DockerPort.TCP(2888), DockerPort.TCP(3888)),
-      name = Some("zookeeper-service"),
-      networks = Seq(zookeeperNetwork)
+      networks = Seq(zookeeperNetwork),
+      reuse = false
     )
   }
 }
 
 class ZookeeperDockerModule[F[_]: TagK] extends ModuleDef {
   make[ZookeeperDocker.Container].fromResource {
-    ZookeeperDocker.make[F]
+    (wr: DockerClientWrapper[F], log: IzLogger, eff: DIEffect[F], effAsync: DIEffectAsync[F]) =>
+      val randomName = s"zookeeper-${UUID.randomUUID().toString.take(8)}"
```

Comment:
Hmm, the names are always random already if you put `name = None`, i think this is not necessary...

Reply:
Yup, but with this, we can access ZK container from Kafka's container via aliases.
Docker does not create name alias for containers with random names.